stable/nextcloud/README.md

```diff
@@ -116,4 +116,4 @@ $ helm install --name my-release -f values.yaml stable/nextcloud
 The [Nextcloud](https://hub.docker.com/_/nextcloud/) image stores the nextcloud data and configurations at the `/var/www/html` paths of the container.
 
 Persistent Volume Claims are used to keep the data across deployments. This is known to work in GCE, AWS, and minikube.
-See the [Configuration](#configuration) section to configure the PVC or to disable persistence.
\ No newline at end of file
+See the [Configuration](#configuration) section to configure the PVC or to disable persistence.
```

Comment:
Thx for the patch. Didn´t you mean 'to enable persistence' here? And I think we need a blank line at the end.

Reply:
Good catch. Reworded a bit. There is also a blank line on the end :)